components/availability/src/init.spec.js

```diff
@@ -2,6 +2,64 @@
 const frozenDateTime = () => new Date(2021, 11, 15, 9, 0, 0);
 const testUser = ["nylascypresstest@gmail.com"];
 
+function buildMockDataObjects(timeSlots, timeslotSize = 15) {
+  const emails = Array.from(
+    new Set(timeSlots.map((slot) => slot.emails).flat()),
+  );
+
+  const availability = {
+    object: "availability",
+    order: emails,
+    time_slots: timeSlots.map((slot) => {
+      return {
+        emails: slot.emails,
+        end: slot.end_time / 1000,
+        end_time: slot.end_time / 1000,
+        object: "time_slot",
+        start: slot.start_time / 1000,
+        start_time: slot.start_time / 1000,
+        status: "free",
+      };
+    }),
+  };
+
+  const bookingOptions = [];
+  for (const timeslot of timeSlots) {
+    const startHours = new Date(timeslot.start_time).getHours(),
```

Comment:
Might be best to use `frozenDateTime` var instead of `new Date` here. I know we're mocking the data, but it will be consistent and prevent and unexpected time-related flakes.

Reply:
I had that initially, but there are a couple of tests that need to test a day before / after `frozenDateTime`